cypress_shared/fixtures/person.json

```diff
@@ -5,7 +5,7 @@
   "nomsNumber": "A9287DY",
   "nationality": "British",
   "religionOrBelief": "Atheist",
-  "status": "InCustody",
+  "status": "InCommunity",
```

Comment:
I was working on a more robust fix for this, as we want someone that's In Custody, so was going to use a different CRN